### ISSUE TITLE
6.0 Feedback:  Dev console error, navigating to section causes "Expected path command" error

### ISSUE THREAD

**Issue by bndo** [Opentrons/opentrons#10536]
### Overview

In 6.0.0-alpha.0, navigating from Devices to Protocols or Labware results in a console error

### Current Behavior

The minified stack trace (not terribly useful)
![image](https://user-images.githubusercontent.com/587066/171218861-a21d81fa-ff91-4307-956c-c84e73539c6b.png)

```
Error: <path> attribute d: Expected path command, "…556H8V0H4.44444Z.83L9.17,16.24L1…".
x @ react-dom.production.min.js:23
rl @ react-dom.production.min.js:208
Ou @ react-dom.production.min.js:251
Eu @ react-dom.production.min.js:250
wu @ react-dom.production.min.js:250
xu @ react-dom.production.min.js:250
hu @ react-dom.production.min.js:243
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:18
Fi @ react-dom.production.min.js:122
Gi @ react-dom.production.min.js:123
Hi @ react-dom.production.min.js:122
Te @ react-dom.production.min.js:292
Jt @ react-dom.production.min.js:73
```

### Steps To Reproduce

1. launch app and toggle dev tools if you don't have them on.
2. filter the dev tools to Errors only.
3. click on Protocols or Devices or Labware

### Operating system

Mac

### Robot setup or anything else?

no robot connected

**Comment by bndo** [comment]
Tested on `6.0.0-alpha.1` 
- Navigating to Devices is clean.
- Navigating to Labware results in one `Error: <path> attribute d: Expected path command...`
- Navigating to Protocols results in three `Error: <path> attribute d: Expected path command...`

**Comment by b-cooper** [comment]
Closed By #10990

**Comment by y3rsh** [comment]
### verified on alpha5
I navigated all over the app and do not see any errors in console.
